lib/europeana/proxy/media.rb

```diff
@@ -96,7 +96,11 @@ def proxy?(env)
       # @return [Hash] rewritten request env
       def rewrite_env(env)
         env['app.record_id'] = env['REQUEST_PATH']
-        edm = Europeana::API.record(env['app.record_id'])['object']
+        env['app.permitted_api_urls'] = ENV['ALLOWED_API_URLS'].split(',').map(&:strip) if ENV['ALLOWED_API_URLS']
+        if env['app.params']['api_url'] && env['app.permitted_api_urls'].include?(env['app.params']['api_url'])
+          Europeana::API.url = env['app.params']['api_url']
```

Comment:
As this is an access restriction, let's 403 here instead, by raising an error then catching it in `#rescue_call_errors` and returning `response_for_status_code(403)`